packages/client/modules/meeting/components/SelectTemplate.tsx

```diff
@@ -66,7 +67,8 @@ const SelectTemplate = (props: Props) => {
   }
   const goToBilling = () => {
     SendClientSegmentEventMutation(atmosphere, 'Upgrade CTA Clicked', {
-      upgradeCTALocation: 'publicTemplate'
+      upgradeCTALocation: 'publicTemplate',
+      meetingType: type as MeetingTypeEnum
```

Comment:
@nickoferrall I think we still have some types defined in `_legacy.graphql`, check the `type ReflectTemplate implements MeetingTemplate {`

Reply:
`type` is `String!` in `_legacy.graphql` but should I directly edit `_legacy.graphql` directly? I'd expect that I can just change `type` from `new GraphQLNonNull(GraphQLString)` to `new GraphQLNonNull(MeetingTypeEnum)` in `MeetingTemplate.ts` and it would update the schema but apparently not

Reply:
@nickoferrall I guess yes, I was unable to find where it is generated, and always edited the file manually. Alternative approach is to refactor specific types on a new approach